File: Java/9.java
/**
 * @no 9
 * @name Palindrome Number
 */
public class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0) return false;
        else if (x == 0) return true;
        else {
            int backup = x;
            int y = 0;
            while (x != 0) {
                y = y * 10 + x % 10;
                x = x / 10;
            }
            return (y == backup);
        }
    }
}

File: Python/9.py
"""
@no 9
@name Palindrome NUmber
"""
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        x = str(x)
        for i in range(len(x) // 2):
            if x[i] != x[len(x) - i - 1]:
                return False
        return True
